### ISSUE TITLE
Suppress yes/no if there are no changes

### ISSUE THREAD

**Issue by ekrengel** [pulumi/pulumi#2616]
The CLI gives you the option to perform the update (yes/no) when there is no changes, which was a bit confusing. We should either suppress the yes/no OR if there is a good reason for it call this out explicitly in docs or alongside the yes/no.

**Comment by pgavlin** [comment]
We made a deliberate choice to retain the yes/no prompt because it is not clear what is the correct action to take by default in this case. IIRC, it is possible for stack outputs to change in a no-op update, but that is likely a bug (the stack outputs should result in an update). I _think_ that barring that issue, we could just skip the no-op update altogether and default to "no" here.

**Comment by nafg** [comment]
How can the outputs change? Can you elaborate?

**Comment by j-martin** [comment]
> We made a deliberate choice to retain the yes/no prompt because it is not clear what is the correct action to take by default in this case. IIRC, it is possible for stack outputs to change in a no-op update, but that is likely a bug (the stack outputs should result in an update). I think that barring that issue, we could just skip the no-op update altogether and default to "no" here.

Yes, but it would be nice to have a flag to override the default behaviour.
